xbmc/filesystem/BlurayFile.h

```diff
@@ -10,18 +10,18 @@
 
 #include "filesystem/OverrideFile.h"
 
-namespace XFILE
+using namespace XFILE;
```

Comment:
No "using namespace" in header files, please!  This can lead to weird compile problems.  

More important: You here also define the namespace for the class defined here! With your change, `CBlurayFile` no longer is in namespace `XFILE`, but it should.

Please refrain  from making changes which are definitely not related to the subject of the commit message. Make a separate commit so such things can easily be spotted and reviewed.